cert-manager/overlays/emea/jerry/secrets/clouddns-dns01-emea-svc-acct.enc.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    creationTimestamp: null
+    name: clouddns-dns01-emea-svc-acct
+stringData:
+    type: ENC[AES256_GCM,data:8NE8E1WT71kglTsBI3kT,iv:AQiDOXkYyymBwcBFTBiig5RrY7WJq1Nf/rXCqc5yE+g=,tag:Xl5OlRgmPFhn0fVLhIjDhw==,type:str]
```

Comment:
Is it correct that every piece of data in this secret is the same as for smaug? Like we call this the `clouddns-dns01-emea-svc-acct` but were literally just using our access credentials for `clouddns-dns01-na-svc-acct`. You can tell this because all the data is the same as well as that the [`client_email`](https://github.com/operate-first/apps/pull/2349/files#diff-d7fac452e80f2b3a2821a0f0b37b2df9072be8a94a07c349de85a2b29520025bR11) references a North American DNS related email. I have no problem with reusing credentials (provided it doesn't introduce a security risk), but in that case shouldn't we just use the existing NA service account?

Reply:
@Gregory-Pereira regarding re-using the service account, maybe this would work, but 1. isn't the service account created on a per cluster basis and 2. I think we have different credentials, the `private_key_id` and `private_key` are different